src/components/payment-options.js

```diff
@@ -13,7 +13,7 @@ var PayPalButton = React.createClass({
     submit: React.PropTypes.array.isRequired,
     name: React.PropTypes.string.isRequired
   },
-  onChange: function() {
+  onClick: function() {
```

Comment:
This is for Paypal.

I switched Stripe button to click, and not change, because change wasn't working.

I figured keep Paypal in sync.